pkg/loadbalancer/bpf_abi.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018-2019 Authors of Cilium
+//
```

Comment:
This filename could be `bpf_abi_test.go` instead?

Reply:
Was this address?

Reply:
As discussed here: https://github.com/cilium/cilium/pull/12415#discussion_r450082690, it seems that it is not possible to change the name to `bpf_abi_test.go`, because there is an error that using cgo in tests is not supported.

Per @tklauser suggestion, I looked at using alignchecker for performing the test, which resulted in https://github.com/cilium/cilium/pull/12436.